src/drivers/lights/blinkm/CMakeLists.txt

```diff
@@ -33,6 +33,7 @@
 px4_add_module(
 	MODULE drivers__blinkm
 	MAIN blinkm
+	STACK_MAIN 1240
```

Comment:
This is smaller than the default.

Reply:
In debugging hardfaults caused by the AIX errata I believe I did a stack check build and set it to what it needed.